build.rs

```diff
@@ -40,6 +40,11 @@ fn main() {
         println!("cargo:rustc-cfg=libc_deny_warnings");
     }
 
+    // Rust > 1.13 supports long array:
+    if rustc_minor_ver > 13 || rustc_dep_of_std {
```

Comment:
Could you use `=>` to keep the consistency with other cfgs?

Reply:
Done.